vembrane/modules/annotate.py

```diff
@@ -41,6 +41,14 @@ def add_subcommmand(subparsers):
         choices=["vcf", "bcf", "uncompressed-bcf"],
         help="Output format.",
     )
+    parser.add_argument(
+        "--backend",
+        "-b",
+        default="pysam",
```

Comment:
As mentioned above, probably use cyvcf2 as the default.

Reply:
cyvcf2 is default now